server/templates/screenshot/commit.html

```diff
@@ -26,12 +26,16 @@
 {% block content %}
   <div class="container">
     <p>
-      <div>Code Link:</div>
-      <div>
-          <a href="https://github.com/datacommonsorg/website/commit/{{ sha }}">
-            https://github.com/datacommonsorg/website/commit/{{ sha }}
-          </a>
-      </div>
+      {% if sha|length == 7 %}
+        <div>Code Link:</div>
+        <div>
+            <a href="https://github.com/datacommonsorg/website/commit/{{ sha }}">
+              https://github.com/datacommonsorg/website/commit/{{ sha }}
+            </a>
+        </div>
+      {% else %}
+        <div><b>Date: {{ sha }}</b></div>
```

Comment:
I think instead of re-using sha to hold the date, add a date field?

Reply:
Use two templates for date and commit pages now